draft-ietf-rtcweb-jsep.xml

```diff
@@ -3671,6 +3665,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           </list>
```

Comment:
For consistency, suggest flipping line 3663 around to [RFC5245], Section 9.3.1.1.